docs/source/installation_conda.rst

```diff
@@ -136,9 +136,9 @@ Now, Install cuDNN v6.0 (skip if you have it already):
     $ CUDNN_TAR_FILE="cudnn-8.0-linux-x64-v6.0.tgz"
     $ wget http://developer.download.nvidia.com/compute/redist/cudnn/v6.0/${CUDNN_TAR_FILE}
     $ tar -xzvf ${CUDNN_TAR_FILE}
-    $ sudo cp -P cuda/include/cudnn.h /usr/local/cuda/include
-    $ sudo cp -P cuda/lib64/libcudnn* /usr/local/cuda/lib64/
-    $ sudo chmod a+r /usr/local/cuda/lib64/libcudnn*
+    $ cp -P cuda/include/cudnn.h /usr/local/cuda/include
```

Comment:
This won't work unless you work under root.  If you do run under root, sudo (if installed) is a noop.

Reply:
in the docker env, sudo doesn't work. so I provided general instructions, if something doesn't work, then people will add sudo themselves.

Reply:
why doesn't it work?  do you need to install `sudo` package?

Reply:
nope, in docker everything is already root, and sudo throws errors `sudo is unknown`

Reply:
which probably means that the `sudo` command is just not installed :)
it's usually possible to run `sudo` or `su` under root on a full installation

Reply:
yeah docker requires specifically installing sudo https://stackoverflow.com/a/25908200 but I didn't feel that was necessary to do. I will add a note for asking people to use sudo if a command fails to run